migrations/v2.js

```diff
@@ -0,0 +1,134 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin, getCourse, testStopWhere, testSuccessWhere } from 'adapt-migrations';
+import _ from 'lodash';
+
+describe('Resources - v2.0.1 to v2.0.2', async () => {
+  let course;
+  whereFromPlugin('Resources - from v2.0.1', { name: 'adapt-contrib-resources', version: '>=2.0.0 <2.0.2' });
+
+  whereContent('Resources - where Resources', async (content) => {
+    course = getCourse();
+    return course?._resources;
+  });
+
+  mutateContent('Resources - add _drawerOrder attribute', async (content) => {
+    course._resources._drawerOrder = 0;
+    return true;
+  });
+
+  checkContent('Resources - check _drawerOrder attribute', async (content) => {
+    const isValid = _.has(course, '_resources._drawerOrder');
+    if (!isValid) throw new Error('Resources - _drawerOrder attribute not updated');
+    return true;
+  });
+
+  updatePlugin('Resources - update to v2.0.2', { name: 'adapt-contrib-resources', version: '2.0.2', framework: '>=2.2.0' });
```

Comment:
```suggestion
  updatePlugin('Resources - update to v2.0.2', { name: 'adapt-contrib-resources', version: '2.0.2', framework: '^2.0.0' });
```
FW [version](https://github.com/adaptlearning/adapt-contrib-resources/blob/v2.0.2/bower.json) should be ^2.0.0 rather than 2.2.0.